doc/guides/backporting-to-release-lines.md

```diff
@@ -44,6 +44,8 @@ cd $NODE_DIR
 git fetch upstream v6.x-staging:v6.x-staging -f
 # Assume we want to backport PR #10157
 git checkout -b backport-10157-to-v6.x v6.x-staging
+# Ensure there are no build artifacts from previous builds
+git clean -xfd
```

Comment:
This is a "dangerous" command, it removes ___everything___ including draft files and IDE metadata.
I'd suggest mark it as optional, and add a warning.
Also AFAIK the only problematic area is `./test` so maybe just:
```
git clean -xdf ./test/
```